packages/component-library/draft/Kaizen/SplitButton/DropdownMenu.tsx

```diff
@@ -24,36 +18,13 @@ export default class DropdownMenu extends React.Component<Props> {
   componentDidMount() {
     document.addEventListener("click", this.handleDocumentClick, false)
     window.addEventListener("resize", this.handleDocumentResize, false)
-    this.positionMenu()
   }
 
   componentWillUnmount() {
     document.removeEventListener("click", this.handleDocumentClick, false)
     window.removeEventListener("resize", this.handleDocumentResize, false)
   }
 
-  positionMenu() {
-    const menu = this.menu
-    if (!this.props.position || !menu) {
-      return
-    }
-    const pos = this.props.position
-    const { innerHeight } = window
```

Comment:
I see there's some logic here that determines if the popover should be above or below, depending on what best fits the browser viewport window. Just confirming that this logic still exists?

Reply:
no, i missed this. Shouldn't being deleted. Added it again but pass the reference until the parent. Thanks for noticing this 🙏